stix_shifter_modules/stix_bundle/stix_transmission/connector.py

```diff
@@ -61,7 +76,16 @@ def ping_connection(self):
             ErrorResponder.fill_error(return_obj, response_txt, ['message'], connector=self.connector)
         return return_obj
 
-    def create_results_connection(self, search_id, offset, length):
+    def create_results_connection(self, search_id, offset, length, metadata=None):
+
+        #METADATA_SAMPLE metadata_result_id = 0
+        #METADATA_SAMPLE if metadata:
+        #METADATA_SAMPLE     metadata_result_id = int(metadata.get('metadata_result_id', 0))
+        #METADATA_SAMPLE else:
+        #METADATA_SAMPLE     metadata = {}
+        #METADATA_SAMPLE metadata_result_id += 1
+        #METADATA_SAMPLE metadata['metadata_result_id'] = metadata_result_id
```

Comment:
Trying to understand this usage-
1. what is the use of `metadata_result_id`?
2. why does it have to be integer?
3. why are you incrementing the id?
4. what if there's a different key other than `metadata_result_id`?

Reply:
it is just a sample, any data can be returned in metadata status. That returned metadata will be provided to future status and results calls.

Reply:
I think, this example would confuse people. Also, I would put example inside dummy connectors instead of stix-bundle `stix_shifter_modules/synchronous_dummy` / `stix_shifter_modules/async_dummy`